### ISSUE TITLE
Build failure triggered by diff-highlight plugin

### ISSUE THREAD

**Issue by brimarq** [mAAdhaTTah/babel-plugin-prismjs#39]
I had a similar build failure to @ReidBraswell 's in #8  
```
Module build failed (from ./node_modules/babel-loader/lib/index.js):
TypeError: /Users/brian/Projects/fcc/responsive-web-design/projects/tech-doc/src/index.js: Cannot read property 'require' of undefined
    at getRequire (/Users/brian/Projects/fcc/responsive-web-design/projects/tech-doc/node_modules/babel-plugin-prismjs/lib/getComponents.js:55:45)
    at /Users/brian/Projects/fcc/responsive-web-design/projects/tech-doc/node_modules/babel-plugin-prismjs/lib/getComponents.js:109:36
    at Array.reduce (<anonymous>)
    at getDependencies (/Users/brian/Projects/fcc/responsive-web-design/projects/tech-doc/node_modules/babel-plugin-prismjs/lib/getComponents.js:107:31)
    at /Users/brian/Projects/fcc/responsive-web-design/projects/tech-doc/node_modules/babel-plugin-prismjs/lib/getComponents.js:109:14
    at Array.reduce (<anonymous>)
    at getDependencies (/Users/brian/Projects/fcc/responsive-web-design/projects/tech-doc/node_modules/babel-plugin-prismjs/lib/getComponents.js:107:31)
    at Array.reduce (<anonymous>)
    at _default (/Users/brian/Projects/fcc/responsive-web-design/projects/tech-doc/node_modules/babel-plugin-prismjs/lib/getComponents.js:149:145)
    at PluginPass.ImportDeclaration (/Users/brian/Projects/fcc/responsive-web-design/projects/tech-doc/node_modules/babel-plugin-prismjs/lib/index.js:27:56)
```
I'm running macOS using: 
- @babel/core - v7.6.4
- prismjs - v1.17.1
- babel-plugin-prismjs - v1.1.1  

and .babelrc config: 
```json
{
  "plugins": [
    ["prismjs", {
        "languages": ["javascript", "css", "markup", "pug", "bash", "diff"],
        "plugins": ["line-numbers", "diff-highlight"],
        "theme": "dark",
        "css": true
    }]
  ]
}
```
Removing "diff-highlight" from the config allows a successful build. With that change, I can manually import the plugin directly into my entry JS file and it will work: 
```js
// index.js
import 'prismjs/plugins/diff-highlight/prism-diff-highlight.css';
import 'prismjs/plugins/diff-highlight/prism-diff-highlight.js';
```
Strange, no? [line-numbers](https://github.com/PrismJS/prism/blob/master/plugins/line-numbers/prism-line-numbers.js) works but [diff-highlight](https://github.com/PrismJS/prism/blob/master/plugins/diff-highlight/prism-diff-highlight.js) crashes this plugin; yet the same `diff-highlight` works via direct `import`? Hmm... 

I have a workable solution for now; maybe these clues will help solve the problem.

**Comment by mAAdhaTTah** [comment]
Please test if this is still an issue in v2.0.0 and comment here if it is. Otherwise, going to close, as we've moved to a new dependency loader, which should solve the problem.

**Comment by qaiken** [comment]
Seems to still be an issue. Seeing a different error now though:
```
Cannot read property 'noCSS' of undefined
```

**Comment by mAAdhaTTah** [comment]
@qaiken Can you provide a reproducible example? I'd be happy to take a look.

**Comment by qaiken** [comment]
"babel-plugin-prismjs": "^2.0.0",
"prismjs": "^1.19.0",
"@babel/core": "^7.3.4",

plugin setting:
```
  [
    'prismjs',
    {
      languages: ['javascript'],
      plugins: ['line-numbers', 'normalize-whitespace', 'diff-highlight'],
      theme: 'default',
      css: true,
    },
  ]
```

**Comment by mAAdhaTTah** [comment]
Fixed in v2.0.1.

**Comment by qaiken** [comment]
thank you! works perfectly
